stats/stats/src/charts/lines/txns_fee.rs

```diff
@@ -31,7 +34,7 @@ const ETHER: i64 = i64::pow(10, 18);
 pub struct TxnsFeeStatement;
 
 impl StatementFromRange for TxnsFeeStatement {
-    fn get_statement(range: Option<Range<DateTimeUtc>>) -> Statement {
+    fn get_statement(range: Option<Range<DateTimeUtc>>, _: &BlockscoutMigrations) -> Statement {
```

Comment:
Similar to `AverageGasPriceStatement` considerations apply

Reply:
2020.970182048s -> ~9.247s (sepolia) that's a win